src/handbook/design/art-requests.md

```diff
@@ -0,0 +1,45 @@
+---
+navTitle: Art Requests
+---
+
+## Art Requests
+
+Across FlowForge it will be a common occurence that we need visual content created. These can be part of development tasks (where you're comfortable doing the development work, but less doing the graphic/illustrative design work) or social/marketing materials.  
+
+Likely places this will be required are:
+
+- Tiles for Blog Posts
+- Tiles & Associated Graphics/Assets for Webinars & AMA Sessions
+- Diagrams for Presentations
+- Social Media Adverts & Graphics
+- YouTube Video Thumbnails
+- Product Illustrations & Graphics
+
+### Creating an Art Request
+
+You can create an issue on any of the FlowForge Repositories that best suits the asset required, e.g:
+
+- [flowforge/flowforge](https://github.com/flowforge/flowforge): In-Product Visuals (e.g. icons, diagrams, pictograms)
+- [flowforge/website](https://github.com/flowforge/website): Markitechture Diagrams, Blog Tiles, Webinar & AMA Tiles
+- [flowforge/content](https://github.com/flowforge/content): Social Media Assets, YouTube Thumbnails Tiles and anything else that doesn't fit above 
+
+When creating an issue, you'll be presented with an option to create an "Art Request" from the Issue Templates:
+
+![Screenshot from GitHub to show the option for creating an Art Request](../images/screenshots/gh-artrequest-template.png "Screenshot from GitHub to show the option for creating an Art Request")
+
+#### Details to Include
+
+Please be sure to include as much detail as possible, but most importantly:
+
+- **Type**: What type of asset is this? Blog Tile? YouTube Thumbnail?
+- **Format**: Image (e.g. `.png`, `.jpg`), Video (e.g. `mp4`), Animation (e.g. `.gif` or Lottie Animtion)
+- **Deadline**: This is defined on the right-side of the issue under the "Art Requests" project, once the issue has been created.
```

Comment:
I can not see how I can add a deadline to this request https://github.com/flowforge/website/issues/545

Reply:
<img width="1295" alt="Screenshot 2023-03-23 at 09 09 34" src="https://user-images.githubusercontent.com/99246719/227155614-e08d974e-194a-450b-8779-fef44dd1f7fb.png">

It's on the right. I mention int he Handbook, the "Art Request" project gets added _after_ you create the issue, so it only becomes an available field then annoyingly.

Reply:
Thanks, I couldn't find it without the screenshot.

Reply:
(Screenshot is also included in the PR/Handbook)